en/docs/try-out/payment-initiation-service-flow.md

```diff
@@ -613,3 +613,336 @@ In this section, you will be generating an access token using the authorization
           "expires_in":3600
        }
       ```
+
+### Payment information/ status retrieval related requests
+
+!!! note
+    All the following endpoints need a user access token to retrieve payment information.
+
+**GET /payments/{payment-product}/{paymentId}**
+
+``` tab="Request"
+curl -X GET 'https://<APIM_HOST>:8243/xs2a/v1/payments/sepa-credit-transfers/4a8ac7c7-d5ee-42cb-8a29-44fc7778bd96' \
+-H 'PSU-IP-Address: 127.0.1.1' \
+-H 'PSU-ID-Type: email' \
+-H 'X-Request-ID: 6bb5c0b0-5f63-4872-b3dc-e4987f58f879' \
+-H 'Date: Mon, 31 Oct 2022 11:59:20 GMT' \
+-H 'PSU-ID: admin@wso2.com' \
+-H 'Signature: keyId="SN=de730ec5733ead1b,CA=1.2.840.113549.1.9.1=#16116d616c7368616e694077736f322e636f6d,CN=OpenBanking Pre-Production Issuing CA,OU=OpenBanking,O=WSO2 (UK) LIMITED,L=COL,ST=WP,C=LK",algorithm="rsa-sha256", headers="digest x-request-id date psu-id",signature=f/ozcQhnzQwGCGNbMe1bdpYMTqECO65uVBrwGCCYGajtilTs9O0wh3cA3Mrdtw2JUreEmSKDFpb0gjNObHCP6wdtOoCZpzU2EKhdhcPFVqkcJfL+9rMrJZU2l+U6sZs+FO7Bc2H/VQnvXXHRjD9UfPkrBLSvftDm1LxodQkei+7QuCfmvltILfynFQ/c/85ITnYqUQzlP6f/LDsWjOgQqXwWTVW7WMOHitLENwGaHNy6EjiIPFZIVYNJdHd51XHbqMNN332QQuE26IRzfUqy2Cus8a94hE8OZ4hx863M7V8YKIfWj3MSJSicSXwCeMClXalphV1NKUixRDKuqds85Q==' \
+-H 'Digest: SHA-256=RBNvo1WzZ4oRRq0W9+hknpT7T8If536DEMBg9hyq/4o= \
+-H 'TPP-Signature-Certificate: <SIGNATURE_CERTIFICATE>' \
+-H 'Authorization: Bearer <APPLICATION_ACCESS_TOKEN>'
+```
+
+``` tab="Response"
+200 OK
+Header: X-Request-ID: 6bb5c0b0-5f63-4872-b3dc-e4987f58f879
+{
+   "debtorAccount": {
+       "iban": "DE12345678901234567890",
+       "currency": "EUR"
+   },
+   "creditorName": "Merchant123",
+   "transactionStatus": "ACCP",
+   "creditorAccount": {
+       "iban": "DE98765432109876543210"
+   },
+   "instructedAmount": {
+       "amount": "123.50",
+       "currency": "EUR"
+   },
+   "remittanceInformationUnstructured": "Ref Number Merchant"
+}
+```
+
+!!! note
+    - This response is constructed by the core banking back end.
```

Comment:
Shall we say "this response should be constructed by the core banking back end"? Let's change the other similar places as well.

Reply:
Fixed via c9dd61f1fa814823ce1ef22598e30d998cefea67.